rpc/api.go

```diff
@@ -186,16 +186,24 @@ func (api *EthereumApi) GetRequestReply(req *RpcRequest, reply *interface{}) err
 			return err
 		}
 		*reply = v
-	case "eth_call":
-		args := new(CallArgs)
-		if err := json.Unmarshal(req.Params, &args); err != nil {
+	case "eth_estimateGas":
+		_, gas, err := api.doCall(req.Params)
+		if err != nil {
 			return err
 		}
 
-		v, err := api.xethAtStateNum(args.BlockNumber).Call(args.From, args.To, args.Value.String(), args.Gas.String(), args.GasPrice.String(), args.Data)
+		// TODO unwrap the parent method's ToHex call
+		if len(gas) == 0 {
+			*reply = newHexData([]byte{})
+		} else {
+			*reply = newHexData(gas)
```

Comment:
For "estimate gas", we should probably output as newHexNum(), since it's a quantity.